resources/css/style.css

```diff
@@ -1,5 +1,5 @@
 html {
-  background-image: linear-gradient(to top, #ccccff 10%, #ccfff2 50%);
+  background-image: url("/bilder/Unbenannt.png");
```

Comment:
Warum ist das Bild unbenannt?

Reply:
Das bild heißt nur so

Reply:
Also ist das gut so?

Reply:
ja das ist richtig, ich hatte das Bild selber gemacht und das dann bei Unbenannt gelassen.

Reply:
Die Frage ist, ob das so gut ist?

Reply:
Nein, weil man es als nicht Wissender, falsch verstehen könnte?

Reply:
Und weil man sich sowas nicht angewöhnen sollte!
Zweckgebundene Namen ;)

Reply:
Okay, hab es in Hintergrund.png unbenannt, gut zu wissen.

Reply:
:+1: